schemas/cloud.yml

```diff
@@ -47,3 +47,13 @@
       example: t2.medium
       description: >
         Machine type of the host machine.
+
+    - name: account.id
+      type: keyword
+      example: 666777888999
+      description: >
+        The cloud account or organization id
+
+        This could be the AWS account id or Google Cloud ORG Id. This should be
```

Comment:
Let's remove the `should be in an extended schema` part here as we don't have that yet.